pkg/runtime/itzo_linux.go

```diff
@@ -32,7 +32,7 @@ type ImagePuller struct {
 }
 
 func (ip *ImagePuller) PullImage(rootdir, name, image string, registryCredentials map[string]api.RegistryCredentials, useOverlayfs bool) error {
-	server, _, err := util.ParseImageSpec(image)
+	server, repo, err := util.ParseImageSpec(image)
 	if err != nil {
```

Comment:
This if statement can be removed. ParseImageSpec will never "fail"

Reply:
nice catch!